### PR TITLE
Another improvements for python 3

### PR DESCRIPTION
**Pull request by Mte90**
I am trying to understand why the autocomplete is not working more.
Seems that padawan is running but the menu not appear when I press tab.
Debugging I discovered few other errors for python3.

### CODE REVIEW

**Comment by Mte90** [comment]
It is not yet finished!

**Comment by Mte90** [comment]
Patch finished, now is working without issues :-D

File: python/padawan.py [hunk 1]
```diff
@@ -1,15 +1,9 @@
...
-    from urllib.parse import urlencode
-    from urllib.request import urlopen
-    from urllib.request import Request
-    from urllib.request import URLError
-except ImportError:
-    from urllib import urlencode
-    from urllib2 import urlopen
-    from urllib2 import Request
-    from urllib2 import URLError
+from urllib.parse import urlencode, quote_plus
```
Comment (mkusher) [inline python/padawan.py:3]: looks like it breaks compatibility with python2, am I right?

Comment (Mte90) [reply python/padawan.py:3]: yes but it was the only solution because, python3 require to encode the data

**Pull request merged**